Makefile

```diff
@@ -1,20 +1,34 @@
 # If CNTR_PROG is undefined, then set it to be podman
 ifndef CNTR_PROG
-	CNTR_PROG = podman
+	CNTR_PROG = docker
 endif
 
-CNTR_USER=gvegayon
-IMAGE_NAME=cfa-epinow2-pipeline-dependencies:latest
+ifndef TAG
+	TAG = local
+endif
+
+IMAGE_NAME=cfa-epinow2-pipeline
+
+deps:
+	$(CNTR_PROG) build -t $(REGISTRY)$(IMAGE_NAME)-dependencies:$(TAG) -f Dockerfile-dependencies
+
+pull:
+	$(CNTR_PROG) pull $(REGISTRY)$(IMAGE_NAME)-dependencies:$(TAG)
```

Comment:
FYI this doesn't work on podman only docker. I'd hardcode it to docker.

Reply:
Swapped everything to docker